src/components/input/validationRules.js

```diff
@@ -0,0 +1,5 @@
+const emailRules = [v => !!v || 'E-mail is required', v => /.+@.+/.test(v) || 'E-mail must be valid',]
+
+const validationRules = {emailRules}
+
+export default validationRules
```

Comment:
Seems cool @Jabuf but I think we can use a very lightweight and useful package for validations. With @mkanzit , we've used is_js in the admin ui. You can check here if you wish: https://github.com/progressive-identity/knowledge-base-admin-ui/blob/main/mixins/validations.js

Our logic is a bit different too. We use the mixins to make it work. You can see how it is imported on line 33 and 36 here : https://github.com/progressive-identity/knowledge-base-admin-ui/blob/main/components/forms/DataCategory.vue
But your solution is quite elegant since we can choose the validation rule by passing props

Reply:
is_js seems great, I'll adapt my code to use it, that way it will be easier and the structure will already be there if someone need to add a new rule.
As for the mixins I'm not too familiar with it but I feel like it seems more adapted for a form directly in the application using it (which will probably have some specific rules also), here since it's a generic input it's seems easier to use props and vuetify ?

Reply:
I agree. Here, to use mixin, I thinlk you'll have to make it a global mixin and therefore, it means all the validations rules will be imported in all component by default. Which is not great. Your solution seems better for your usecase ;)

Reply:
I've updated the code to use is_js.